Moq.AutoMock/Moq.AutoMock.csproj

```
@@ -6,7 +6,7 @@
     <Copyright>Copyright © $([System.DateTime]::UtcNow.ToString("yyyy"))</Copyright>
     <NeutralLanguage>en-US</NeutralLanguage>
 
-    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">2.1.0</VersionPrefix>
+    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">2.1.1</VersionPrefix>

```

This should be a minor version rev. https://semver.org/
```suggestion
    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">2.2.0</VersionPrefix>
```